test/jdk/java/awt/Toolkit/AWTEventListenerProxyTest/AWTEventListenerProxyTest.java

```diff
@@ -23,7 +23,7 @@
 
 /*
```

Comment:
```suggestion
/*
 * @test
 * @bug 4290704 8357598
 * @summary Test use of AWTEventListenerProxyTest class
 */
```

Most of the test follow this pattern for jtreg tags. It would be good if you use it too.